app/views/static_pages/home.html.erb

```diff
@@ -6,8 +6,6 @@
   <%= link_to "Tools & Resources", tools_resources_path %>
 </div>
 
-<div class="map"></div>
-
 <div class="embed-container">
-  <%= embed("https://www.youtube.com/watch?v=ZzQ3eBerHfM") %>
+  <%= YoutubeHelper.embed("https://www.youtube.com/watch?v=ZzQ3eBerHfM") %>
 </div>
```

Comment:
Cool, I see it is clearer what the embed method is doing now that its a class method!